utils/dbus/Makefile

```diff
@@ -78,27 +80,37 @@ define Package/dbus/config
   source "$(SOURCE)/Config.in"
 endef
 
-CMAKE_OPTIONS += \
-	-DDBUS_SYSTEM_SOCKET=/var/run/dbus/system_bus_socket \
-	-DDBUS_SESSION_SOCKET_DIR=/tmp \
-	-DDBUS_BUILD_TESTS=OFF \
-	-DDBUS_LINUX=ON \
-	-DDBUS_DISABLE_ASSERT=ON \
-	-DBUS_ENABLE_STATS=OFF \
-	-DDBUS_ENABLE_CONTAINERS=OFF \
-	-DENABLE_SYSTEMD=OFF \
-	-DDBUS_BUS_ENABLE_SYSTEMD=OFF \
-	-DHAVE_SYSTEMD=OFF \
-	-DDBUS_WITH_GLIB=OFF \
-	-DDBUS_ENABLE_VERBOSE_MODE=OFF \
-	-DDBUS_DISABLE_CHECKS=ON \
-	-DDBUS_BUILD_X11=OFF \
-	-DDBUS_ENABLE_DOXYGEN_DOCS=OFF \
-	-DDBUS_ENABLE_QTHELP_DOCS=OFF \
-	-DDBUS_ENABLE_XML_DOCS=OFF
+CONFIGURE_ARGS += \
+	--disable-maintainer-mode \
+	--disable-developer \
+	--enable-debug=no \
+	--enable-shared \
+	--disable-static \
+	--disable-verbose-mode \
+	--disable-asserts \
+	--disable-xml-docs \
+	--disable-doxygen-docs \
+	--disable-ducktype-docs \
+	--disable-selinux \
+	--disable-apparmor \
+	--disable-libaudit \
+	--enable-inotify \
+	--disable-kqueue \
+	--disable-console-owner-file \
+	--disable-systemd \
+	--disable-tests \
+	--disable-code-coverage \
+	--disable-x11-autolaunch \
+	--with-session-socket-dir=/tmp \
+	--with-system-socket=/var/run/dbus/system_bus_socket \
+	--with-system-pid-file=/var/run/dbus.pid \
+	--with-dbus-user=root \
+	--without-x \
+	--enable-qt-help=no \
+	--disable-xml-docs
 
 ifeq ($(CONFIG_DBUS_VERBOSE),y)
-  CMAKE_OPTIONS += -DDBUS_ENABLE_VERBOSE_MODE=ON
+  CONFIGURE_ARGS += --disable-verbose-mode
```

Comment:
This was probably meant to be `--enable-verbose-mode`?

Reply:
I guess you're right. I'll have to look at that again. Thanks for the tip.

Reply:
Fixed with commit https://github.com/openwrt/packages/commit/2cbe230b6a65fd00c04cd528f6e5d4651218e1f8 :+1: